notebooks/official/workbench/exploratory_data_analysis/explore_data_in_bigquery_with_workbench.ipynb

```diff
@@ -945,7 +870,7 @@
       },
       "source": [
         "### Function to generalize event counts bar plot creation\n",
-        "Suppose that you like the previous interactive bar plot of event counts, and want to look at it for other categorical variables in the data like those related to device, user characteristics, or other fields. A nice to way generalize this is to create a function that gets event counts data from BigQuery for any single variable and then creates an interactive bar plot like the one above. Run the following cell to create such a function."
+        "Suppose that you like the previous interactive bar plot of event counts, and want to look at it for other categorical variables in the data like those related to device, user characteristics, or other fields. A nice way to generalize this is to create a function that gets event counts data from BigQuery for any single variable and then creates an interactive bar plot like the one above. Run the following cell to create such function."
```

Comment:
What does "it" refer to? another interactive bar plot? Please clarify what "it" refers to.  ("...and want to look at it for other categorical variables..")

Reply:
here  , it refers to previous interactive bar plot